frontend/src/constants/Stringconstant.ts

```diff
@@ -0,0 +1,13 @@
+export const USERNAME_PLACEHOLDER = "Username"
```

Comment:
Don't even already have constants file in the frontend? if we have lets use it. And naming of the file is not matching the guidelines